utils/python/CIME/XML/namelist_definition.py

```diff
@@ -405,5 +409,5 @@ def get_default_value(self, item, attribute=None):
         if attribute is not None:
             all_attributes.update(attribute)
 
-        value = self.get_value_match(item.lower(), all_attributes, True)
+        value = self.get_value_match(item.lower(), all_attributes, True, )
```

Comment:
Fixed the comma. 
I'd like to defer creating common library code to a future PR.